src/exporter/DaemonMetricCollector.cc

```diff
@@ -80,7 +80,22 @@ std::string boost_string_to_std(boost::json::string js) {
 
 std::string quote(std::string value) { return "\"" + value + "\""; }
 
-bool is_hyphen(char ch) { return ch == '-'; }
+void promethize(std::string &name) {
```

Comment:
Let's add "Must be kept in sync with promethize() in src/pybind/mgr/prometheus/module.py" comment here and a corresponding comment in `src/pybind/mgr/prometheus/module.py`.